tests/test_parsl_executors.py

```diff
@@ -0,0 +1,155 @@
+"""
```

Comment:
I would move these tests to the `tests/test_backend_parsl.py` or rename this test something along the lines of `tests/test_backend_parsl_executors.py`, so as to have clear that this stuff logically belongs to the runner backend subsystem and in particular the parsl backend.

Reply:
I'm going with `test_backend_parsl_executors`, because it is reasonably different from `test_backend_parsl.py` (the new one uses actual objects instead of mocks, it uses the db, it uses `process_worfklow`).